SU-SOE/stanford_news
# added copy-webpack-plugin for static assets in js files

# READY FOR REVIEW

# Summary
- Static svg assets in js files copied to build so that they are available for front end theming.

# Review Needed By (Date)
- ASAP

# Urgency
- Critical to finishing news node

# Steps to Test

1. In working env, navigate to stanford_news dir
2. checkout this branch
3. run npm install
4. run npm run build
5. verify that any svg in lib/assets/svg get copied into dis/assets/svg
6. create or go to a news node and view that the social media icons are present.

# Affected Projects or Products
- leleandd8

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/D8CORE-1661
- @jenbreese @sherakama
